cmd/namespace/namespace.go

```diff
@@ -43,16 +44,31 @@ func Namespace(ctx context.Context) *cobra.Command {
 				return err
 			}
 
-			err := RunNamespace(ctx, namespace)
+			err := RunNamespace(ctx, namespace, oktetoURL)
 			analytics.TrackNamespace(err == nil)
 			return err
 		},
 	}
+	cmd.Flags().StringVarP(&oktetoURL, "url", "u", "", "Okteto URL (optional)")
```

Comment:
I see this just as a mere shortcut and it mainly helps reduce the steps to bring users to development from the UI. Currently, the jump from deploy to development takes around 5 - 6 steps when onboarding a new user, back and forth from the UI to the CLI. In the end, the screen to guide users to developments looks like this:

<img width="718" alt="Screenshot 2020-06-16 at 19 38 15" src="https://user-images.githubusercontent.com/237819/84808433-f97a7300-b008-11ea-95e1-c176e7a3a723.png">

On a big screen, you even have to scroll to see the `okteto up` which is the main command in this sequence... And I really believe we need to do something to reduce this friction.

Users starting the process from the UI can benefit from copy/pasting the command with all the needed information to start the process. Just like I do when I'm going to clone a Github repo and don't care about the URL, or when configuring a GKE cluster using the `gcloud` CLI... to name a couple of examples.

I think a good start is to launch the `login` process whenever it is needed to remove an extra step. In Okteto Cloud this parameter is not even necessary. It will only be needed if you copy/paste the command from the UI in a Okteto Enterprise installation...

So, I do feel strongly about having login here for this reason. I am however open check the idea of prompting the user asking for the URL, like you suggest. But think that this is just optional for the flow initiated from the UI.

Reply:
I'm also open for ideas to simplify the path to development if you have any.

As you know, I'd even go for a more aggressive simplification: a single command that helps the user all the way from the CLI. Which will mean to automatically log in, switch namespaces, and clone (if needed) before doing the `up`. But I know you guys are not very fond of this idea...

Reply:
I don't think that bundling commands is a way to simplify the use of okteto. Pretty much every developer tool I can think of went in the direction of having single commands: docker, heroku, terraform, pulumi, snyk. 

Bundling commands adds complexity to the surface area of the command too. Now the developer needs to understand that `okteto namespace` can also login if she's not logged in, so she might want to pass the URL in case she's not.  These companies and tools have spent a lot of effort and research on this topic.

Reply:
If you want to keep the behavior, that's fine, but I would not expose it via  a parameter.

Reply:
@rberrelleza how would you expose it?

Reply:
Use runs `okteto namespace $NAME` like today. If the user is not logged in to okteto cloud, it gets a prompt, where we ask if the user wants to login, ask for the URL (defaulting to cloud), and we do the login action and then if successful, the namespace activation.

Reply:
@rberrelleza LGTM. @rlamana ?

Reply:
ok, sounds good to me too